yarn.lock

```diff
@@ -7787,7 +7787,7 @@ supports-color@1.2.0:
   version "1.2.0"
   resolved "https://registry.yarnpkg.com/supports-color/-/supports-color-1.2.0.tgz#ff1ed1e61169d06b3cf2d588e188b18d8847e17e"
 
-supports-color@3.1.2, supports-color@^3.1.0, supports-color@^3.1.1:
```

Comment:
Yeah, I don't know what the deal is with these. I had to `yarn install` to get the correct version of vets-json-schema, but that also updated the lockfile...Usually I just ignore it, but...I guess I didn't this time. I'll change that back.